docs/administration/csv-mappers.md

```diff
@@ -36,6 +36,38 @@ You can do the same for all the relationships between entities that might be def
 
 ![New representation](assets/csv-mappers-rep-1.png)
 
+## Create a new reference representation (label, author, markings,..)
+
+If you need to map an entity in your CSV mapper with **ref attributes as author, markings, label, external references or kill chain phase**, you must add a new entity representation. 
+
+
+First in the field, you must see no options available: 
+
+![csv](assets/csv-mapper-author-no-options.png)
+
+
+According to the reference, in this array, you will find the representation you need to configure corresponding to the appropriate attribute:
+
+| Attribute reference | Entity representation | 
+|:--------------------|:----------------------|
+| author              | Individual            |
```

Comment:
I suppose it works also for Organization or System too (see what type of entity you can set in author of a report for instance).